dist/tools/ci/print_toolchain_versions.sh

```diff
@@ -38,13 +38,27 @@ get_kernel_info() {
     uname -mprs
 }
 
+have_lsb_release() {
+    lsb_release >/dev/null 2>&1
+    if [ $? -eq 127 ] ; then
+        return 1
+    else
+        return 0
+    fi
+}
+
 get_os_info() {
     local os="$(uname -s)"
     local osname="unknown"
     local osvers="unknown"
     if [ "$os" = "Linux" ]; then
-        osname="$(cat /etc/os-release | grep ^NAME= | awk -F'=' '{print $2}')"
```

Comment:
why not have `if [ -f /etc/os-release ]; then` before that and use `lsb_release` otherwise, and if that's not there either then its `Unknown OS` (or so).

Reply:
Then we have two ways of getting the release, for no gain. I prefer to just ignore /etc/os-release and go with the LSB tool instead.

Reply:
It would also make https://github.com/RIOT-OS/riotdocker/pull/63 unnecessary.

Reply:
The LSB tools are already a requirement of many packages designed to be more or less distribution independent, it is a small tool and is easily available. In centos it is the `redhat-lsb-core`.

IMO not using `lsb_release` is preferable to having two systems to detect the version. If the issue is with riotdocker, it already has like 3Gb, so there is really no impact in an additional package.

Reply:
Size isn't the matter here. Just the required redeployment within the CI for a local problem of yours ;-).

Reply:
> Just the required redeployment within the CI

We don't need to redeploy the CI _now_. https://github.com/RIOT-OS/riotdocker/pull/63 can be merged, wait until enough changes pile up and when the new image is finally deployed, then this PR can go in.